modules/graceful-shutdown.adoc

```diff
@@ -19,92 +18,74 @@ You can shut down a cluster until a year from the installation date and expect i
 +
-[source,terminal]
-----
-sh-4.4# chroot /host
-----
-
-... Check the kubelet client certificate expiration date by running the following command:
+. Mark all the nodes in the cluster as unschedulable. You can do this from your cloud provider's web console, or run the following loop:
 +
 [source,terminal]
 ----
-sh-5.1# openssl x509 -in /var/lib/kubelet/pki/kubelet-client-current.pem -noout -enddate
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm cordon ${node} ; done
 ----
 +
 .Example output
-[source,terminal]
-----
-notAfter=Jun  6 10:50:07 2023 GMT
-----
 
-... Check the kubelet server certificate expiration date by running the following command:
-+
 [source,terminal]
 ----
-sh-5.1# openssl x509 -in /var/lib/kubelet/pki/kubelet-server-current.pem -noout -enddate
+ci-ln-mgdnf4b-72292-n547t-master-0
+node/ci-ln-mgdnf4b-72292-n547t-master-0 cordoned
+ci-ln-mgdnf4b-72292-n547t-master-1
+node/ci-ln-mgdnf4b-72292-n547t-master-1 cordoned
+ci-ln-mgdnf4b-72292-n547t-master-2
+node/ci-ln-mgdnf4b-72292-n547t-master-2 cordoned
+ci-ln-mgdnf4b-72292-n547t-worker-a-s7ntl
+node/ci-ln-mgdnf4b-72292-n547t-worker-a-s7ntl cordoned
+ci-ln-mgdnf4b-72292-n547t-worker-b-cmc9k
+node/ci-ln-mgdnf4b-72292-n547t-worker-b-cmc9k cordoned
+ci-ln-mgdnf4b-72292-n547t-worker-c-vcmtn
+node/ci-ln-mgdnf4b-72292-n547t-worker-c-vcmtn cordoned
 ----
 +
-.Example output
-[source,terminal]
+. Evacuate the pods using the following method:
+[source,yaml]
++
 ----
-notAfter=Jun  6 10:50:07 2023 GMT
+$ for node in $(oc get nodes -l node-role.kubernetes.io/worker -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm drain ${node} --delete-emptydir-data --ignore-daemonsets=true --timeout=15s ; done
 ----
-
-... Exit the debug session.
-
-... Repeat these steps to check certificate expiration dates on all control plane nodes. To ensure that the cluster can restart gracefully, plan to restart it before the earliest certificate expiration date.
-
-. Shut down all of the nodes in the cluster. You can do this from your cloud provider's web console, or run the following loop:
++
+. Shut down all of the nodes in the cluster. You can do this from your cloud provider’s web console, or run the following loop:
 +
 [source,terminal]
 ----
-$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done <1>
+$ for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do oc debug node/${node} -- chroot /host shutdown -h 1; done
 ----
-<1> `-h 1` indicates how long, in minutes, this process lasts before the control-plane nodes are shut down. For large-scale clusters with 10 nodes or more, set to 10 minutes or longer to make sure all the compute nodes have time to shut down first.
-+
 .Example output
+[source,terminal]
+
```

Comment:
Remove the blank line here. There's no space between `[source,terminal]` and the codeblock.